<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager</class>
 <widget class="QWidget" name="Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>欢迎、管理员</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>79</y>
     <width>591</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page_menu">
    <widget class="QPushButton" name="book_manage">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>81</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>图书管理</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reader_manage">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>81</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>读者信息一览</string>
     </property>
    </widget>
    <widget class="QPushButton" name="change_password">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>修改密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="data_manage">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>120</y>
       <width>81</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>系统数据管理</string>
     </property>
    </widget>
    <widget class="QPushButton" name="the_feedback">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>290</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>查看反馈</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_book">
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>571</width>
       <height>261</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>书名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>作者</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>分类</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ISBN</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书籍唯一表示码</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>当前是否在馆</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="book_serch_by_name">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>按书名查找</string>
     </property>
    </widget>
    <widget class="QPushButton" name="book_serch_by_cata">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>按类别查找</string>
     </property>
    </widget>
    <widget class="QPushButton" name="book_add">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>增加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="book_change">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="book_delete">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="book_quit">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_reader">
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>61</y>
       <width>581</width>
       <height>281</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>身份证号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>电话</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>欠费金额</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="reader_quit">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_data">
    <widget class="QPushButton" name="data_databackup">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>73</y>
       <width>111</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>数据备份</string>
     </property>
    </widget>
    <widget class="QPushButton" name="data_datarecover">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>73</y>
       <width>121</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>数据恢复</string>
     </property>
    </widget>
    <widget class="QPushButton" name="data_quit">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_feed_back">
    <widget class="QPushButton" name="feedback_quit">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>290</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>551</width>
       <height>271</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>反馈</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_change_pass">
    <widget class="QLineEdit" name="old_pass_line">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_pass_line">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="conflig_new_pass_line">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>原密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>新密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>283</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>请确认新密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="change_change">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>190</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>修改密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="change_quit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>230</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>141</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string>LOGO</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
